src/osdag/design_report/reportGenerator_latex.py

```diff
@@ -409,26 +411,25 @@ def save_latex(self, uiObj, Design_Check, reportsummary, filename, rel_path, Dis
 
         if does_design_exist and sys.platform != 'darwin' and Disp_3d_image != '':
             doc.append(NewPage())
-            Disp_top_image = "/ResourceFiles/images/top.png"
```

Comment:
This this the part that needs to be turned into temp/cache files. We don't want these images to exist in the installation.